### PR TITLE
Updated copyright (it's 2015!)

### PR DESCRIPTION
**Pull request by SamuelMarks**

### CODE REVIEW

**Comment by deis-admin** [comment]
Thanks for the contribution! Please ensure your commits follow our [style guide](http://docs.deis.io/en/latest/contributing/standards/#commit-style-guide). This code will be tested once a Deis maintainer reviews it.

File: LICENSE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright 2013, 2014 Engine Yard, Inc.
+Copyright 2013, 2014, 2015 Engine Yard, Inc.
```
Comment (eMxyzptlk) [inline LICENSE]: I think it makes sence now to have it as a range `2013 - 2015`

Comment (SamuelMarks) [inline LICENSE]: Sure thing, done (see new commit).

**Comment by bacongobbler** [comment]
LGTM! Would you mind squashing/amending your commits to follow our style guide as @deis-admin pointed out? Thanks!

**Comment by bacongobbler** [comment]
ping

**Comment by SamuelMarks** [comment]
@bacongobbler apologies, got caught up in work. Squashed into one commit.

**Comment by carmstrong** [comment]
@SamuelMarks Thanks for taking the time to update this! Admittedly, we're not sure on what the copyrights should look like, and how to list the years. This is especially complicated because the original owner was OpDemand, and now we're pat of Engine Yard. We're going to ask our lawyers on how to handle this, so we're going to close the PR for now.

Thanks again!

**Pull request closed without merging**